### PR TITLE
Show schemas in streamlit

### PR DESCRIPTION
**Pull request by deeplook**
### Description

This shows all tables for all schemas (one per `dlt` source).

### Related Issues

- Fixes parts of https://github.com/dlt-hub/dlt/issues/459

### Additional Context

This addresses the "Pipeline Info" section. Feedback on the role of the "default_schema" is welcome, as well as hint for changes needed in the "Load Info" section.

### Screenshot

streamlit_schemas

### CODE REVIEW

File: dlt/helpers/streamlit_helper.py [hunk 1]
```diff
@@ -225,16 +227,28 @@ def _query_data(query: str, chunk_size: int = None) -> pd.DataFrame:
...
-    else:
-        schema = pipeline.default_schema
-    st.title(f"Available tables in {schema.name} schema")
-
-    for table in schema.data_tables():
+    # Not needed anymore(?)
+    # if schema_name:
+    #     schema = pipeline.schemas[schema_name]
+    # else:
+    #     schema = pipeline.default_schema
```
Comment (burnash) [inline dlt/helpers/streamlit_helper.py:234]: Yes, it's not needed anymore. Please remove it.

**Review by burnash — CHANGES_REQUESTED** [review]
Looks good to me. Please remove the commented part and it's good to go.

**Comment by burnash** [comment]
@deeplook Thank you very much for the contribution!

File: dlt/helpers/streamlit_helper.py (hunk 1 above)
Comment (deeplook) [reply dlt/helpers/streamlit_helper.py]: Done.

**Review by burnash — APPROVED** [review]

**Pull request merged**